### DISCUSSION TITLE
multi blog

### DISCUSSION THREAD

**Discussion by dgoosens** [squidfunk/mkdocs-material#6280] [Q&A] [answered]
### Description

hi Martin

wondering if it would be possible to have mutiple blogs on the same instance...  
would like to dissociate my `dev` posts from other posts

categories, tags etc can all be common...  
would just like to have two distinct entries in my navigation pointing to a distinct list of articles

### Related links

n/a

### Use Cases

one blog for dev talk   
another one to post non-dev related posts 

### Before submitting

- [X] I have read and followed the [change request guidelines](https://squidfunk.github.io/mkdocs-material/contributing/requesting-a-change/).
- [X] I have verified that [my idea is a change request and not a bug report](https://squidfunk.github.io/mkdocs-material/contributing/requesting-a-change/#its-not-a-bug-its-a-feature).
- [X] I have ensured that, to the best of my knowledge, [my idea will benefit the entire community](https://squidfunk.github.io/mkdocs-material/contributing/requesting-a-change/#benefit-for-the-community).
- [X] I have included relevant links to [the documentation](https://squidfunk.github.io/mkdocs-material/), related [issues](https://github.com/squidfunk/mkdocs-material/issues), and [discussions](https://github.com/squidfunk/mkdocs-material/discussions) to underline the need for my idea.

**Comment by squidfunk** [comment, marked as answer]
Yes, this is already possible! You can use two instances of the [blog plugin](https://squidfunk.github.io/mkdocs-material/plugins/blog) next to each other as long as they are hosted in different directories. Something like the following should work:

``` sh
.
├─ docs/
│  ├─ blog-1/
│  │  ├─ posts/
│  │  └─ index.md
│  └─ blog-2/
│     ├─ posts/
│     └─ index.md
└─ mkdocs.yml
```

Then, in `mkdocs.yml`:

``` yaml
plugins:
  - blog:
      blog_dir: blog-1
  - blog:
      blog_dir: blog-2

nav:
  - Blog-1:
    - blog-1/index.md
  - Blog-2:
    - blog-2/index.md
```

**Comment by dgoosens** [comment]
you rock @squidfunk !!  
will test that tonight

thanks !

**Comment by dgoosens** [comment]
it works !!!   
![carlton-dancing-oh-yeah](https://github.com/squidfunk/mkdocs-material/assets/1250047/d1ebd0ad-ad02-4021-8e78-989c79142a7f)


thanks a lot Martin !
